Automattic/wp-calypso
# Help Center: Check and show notice when zendesk is inaccessible

Fixes https://github.com/Automattic/wp-calypso/issues/94698

## Proposed Changes

* When third-party cookies are disabled we show the chat option, but on click to open it we show broken chat.
* For consistency this PR checks if third-party are disabled and will disable the chat button and display a notice.

## Why are these changes being made?


* To offer a more consistent experience rather than rendering broken chat.

## Testing Instructions



* In firefox set these settings: ![Screenshot(46)](https://github.com/user-attachments/assets/4ff063db-ecac-46c9-9b53-392086e99aef)
* Build this branch and access the help center with user that has chat level support.
* There should be a notice displayed on the contact page and the chat button should be disabled.

## Pre-merge Checklist
